subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/execution/ExecuteActionsTaskExecuterTest.groovy

```diff
@@ -198,7 +247,7 @@ class ExecuteActionsTaskExecuterTest extends Specification {
         then:
         1 * standardOutputCapture.stop()
         then:
-        noMoreInteractions()
+        //noMoreInteractions()
```

Comment:
I think this should be enabled again, right?

Reply:
Oops, fixed.